jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/MultipleInputsTests.java

```diff
@@ -0,0 +1,262 @@
+package io.github.epam.bootstrap.tests.complex;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import static com.epam.jdi.light.ui.bootstrap.BootstrapUtils.isElementInViewPort;
+import static io.github.com.StaticSite.bsPage;
+import static io.github.com.pages.BootstrapPage.multipleInputs;
+import static io.github.com.pages.BootstrapPage.redButton;
+import static io.github.epam.bootstrap.tests.BaseValidations.baseValidation;
+import static io.github.epam.states.States.shouldBeLoggedIn;
+import static io.github.epam.test.data.InputDataHelper.inputDataAsMap;
+import static org.hamcrest.Matchers.containsString;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class MultipleInputsTests extends TestsInit {
+
+    private static Map<Integer, String> inputData;
+    private static List<String> inputDataList;
+
+    @BeforeClass
+    public void beforeClass() {
+        inputData = inputDataAsMap(2);
+        inputDataList = new ArrayList<String>(inputData.values());
+    }
+
+    @BeforeMethod
+    public void before() {
+        shouldBeLoggedIn();
+        bsPage.shouldBeOpened();
+        multipleInputs.setAllValues(inputDataList);
+    }
+
+    @Test
+    public void getTextTest() {
+        int index = 1;
+
+        String name = multipleInputs.getText(index);
+        assertEquals(name, inputData.get(index));
+
+        String surname = multipleInputs.getText("#mi-i-2");
+        assertEquals(surname, inputData.get(2));
+
+        String text = multipleInputs.getText();
+        assertEquals(text, inputData.get(1));
+    }
+
+    @Test
+    public void getTextAllTest() {
+        assertEquals(multipleInputs.getAllTexts(), inputDataList);
+    }
+
+    @Test
+    public void getValueTest() {
+        int index = 1;
+
+        String name = multipleInputs.getValue(index);
+        assertEquals(name, inputData.get(index));
```

Comment:
multipleInputs.get(index).is().value(name) or 
multipleInputs.get(index).is().name(name)